mlflow/utils/environment.py

```diff
@@ -10,7 +10,7 @@
 
 
 def _mlflow_conda_env(path=None, additional_conda_deps=None, additional_pip_deps=None,
-                      additional_conda_channels=None):
+                      additional_conda_channels=None, install_mlflow=True):
```

Comment:
Do we need `install_mlflow`? It's only used in tests. We could just default it to true.

Reply:
It does default to true? Or you mean we could just remove it altogether?  In that case we would need to create a separate version for tests.